sunpy/map/map_factory.py

```diff
@@ -256,6 +260,7 @@ def __call__(self, *args, **kwargs):
         # Hack to get around Python 2.x not backporting PEP 3102.
         composite = kwargs.pop('composite', False)
         cube = kwargs.pop('cube', False)
```

Comment:
Should we throw a deprecation warning here specifically for the keyword argument? The one for constructing `MapCube` isn't very constructive for teaching people to change from `Map(cube=True)` to `Map(sequence=True)`.